pkg/users/account-create-dialog.js

```diff
@@ -29,10 +39,17 @@ import { HelpIcon } from '@patternfly/react-icons';
 
 const _ = cockpit.gettext;
 
-function AccountCreateBody({ state, errors, change }) {
+function get_default_home_dir(base_home_dir, user_name) {
+    return base_home_dir && user_name
+        ? base_home_dir + '/' + user_name
+        : "";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230223-104158-28786987-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)